### PR TITLE
Improve Evaluation Metrics and HTML minifier

### PR DESCRIPTION
**Pull request by Neeraj319**
## Pull Request Summary

This PR includes the following changes:

### Changes in `htmlMinifier.js`
- Added a custom attribute `data-supported-primitives` to the minified HTML.
  - This attribute helps the LLM (Language Model) identify if an element is `clickable`, `hoverable`, etc.
- Added helper functions:
  - `isClickable`: Determines if an element is clickable.
  - `isHoverable`: Determines if an element is hoverable.
  - `canInputText`: Determines if an element can accept text input.

### Updates in the `eval/` folder
- Updated evaluations for the following sites:
  - `bigbasket`
  - `flipkart`
  - `makemytrip`
- Added an evaluation for `99bikes.com`.
- Removed evaluation for `jackphelanchevy.info` due to constant blocking issues.

### Log Line Improvements
- Improved log messages for better clarity.

### CODE REVIEW

**Comment by Neeraj319** [comment]
#5 #6

File: meta/htmlMinifier.js [hunk 1]
```diff
@@ -210,6 +212,141 @@ function getTrimmedAttributes(element) {
...
+
+/**
+ * Determines if an element is clickable.
+ * 
+ * @param {HTMLElement} element - The DOM element to check.
+ * @returns {boolean} True if the element is considered clickable, false otherwise.
+ */
+function isClickable(element) {
+	if (!element) return false;
+
```
Comment (CaffeineDuck) [inline meta/htmlMinifier.js:261]: Add a `canInputText` check within `isClickable` and return `true` if it is **canInputText**

File: meta/htmlMinifier.js [hunk 2]
```diff
@@ -396,6 +539,14 @@ function isValidLocator(locator) {
 		return false;
 	}
 }
+function processChildren(children) {
```
Comment (CaffeineDuck) [inline meta/htmlMinifier.js:542]: Remove this function.

**Review by CaffeineDuck — APPROVED** [review]
lgtm

**Pull request merged**